ddp-workspace/projects/ddp-sdk/src/lib/models/activity/MailAddressBlock.ts

```diff
@@ -5,6 +5,8 @@ export class MailAddressBlock extends ActivityBlock {
     public displayNumber: number | null;
     public titleText: string | null;
     public subtitleText: string | null;
+    public requireVerified: boolean;
```

Comment:
This is not doing anything, right?

Reply:
We'll need to add it sooner or later right? I mean I can remove it here and someone can add it in later.

Reply:
It's fine. Just checking that not an oversight or incomplete.